src/js/components/chart/utils-normalization.js

```diff
@@ -0,0 +1,219 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
+
+import zip from 'lodash/zip';
```

Comment:
We try to avoid pulling in extra dependencies wherever possible. For this set:
flatten could be replaced with `{array}.reduce((a, b) => a.concat(b))`
isInteger could be replaced with a [Polyfil](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger)
sortedIndexOf could be replaced by indexOf as most data sets will be small enough to not notice the performance boost. We are already traversing the data set linearly in `getXValues`, `getMinVal`, and `getMaxVal`.
I'm still understanding how `zip` is being used.